docbot/main.go

```diff
@@ -607,26 +31,8 @@ func main() {
 			logger.Fatalln("Action type is not string")
```

Comment:
Perhaps we can remove the `case` branch above:

```go
	case "issues":
		logger.Infoln("@EventName is issues")
```

Since this action is only intended to be used for PRs.